GLOSSARY.md

```diff
@@ -0,0 +1,31 @@
+<dl>
+  <dt>GPR</dt>
+  <dd>gaussian process regression</dd>
+
+  <dt>SVGP</dt>
+  <dd>stochastic variational inference for Gaussian process models</dd>
+
+  <dt>[] shape constructions</dt>
```

Comment:
```suggestion
  <dt>`[...]` shape annotations</dt>
```

Reply:
@st-- , it is slightly misleading because later we say that the ellipsis sign `...` denotes leading dimensions.

Reply:
@st-- , but I agree that empty brackets don't tell much as well.